examples/cli/test/starknet.test.ts

```diff
@@ -1,35 +1,24 @@
 import { createVcr } from "@apibara/indexer/testing";
+import { describe, expect, it } from "vitest";
 
-import { drizzle } from "drizzle-orm/pglite";
-import { beforeAll, describe, expect, it } from "vitest";
+import createIndexer from "@/indexers/2-starknet.indexer";
 
-import { createIndexer } from "@/indexers/2-starknet.indexer";
 import { starknetUsdcTransfers } from "@/lib/schema";
-import * as schema from "@/lib/schema";
-
-import { migratePglite } from "./helper";
 
 const vcr = createVcr();
-
-const database = drizzle({
-  schema,
-  connection: {
-    dataDir: "memory://starknet",
-  },
-});
+const connectionString = "memory://starknet";
 
 describe("Starknet USDC Transfers indexer", () => {
-  beforeAll(async () => {
-    await migratePglite(database);
-  });
-
   it("should work", async () => {
-    const indexer = createIndexer({ database });
+    const indexer = createIndexer({ connectionString });
 
     await vcr.run("starknet-usdc-transfers", indexer, {
       fromBlock: 800_000n,
       toBlock: 800_005n,
     });
+
+    const database = vcr.getDrizzleDB();
+
```

Comment:
```suggestion
    const testResult = await vcr.run("starknet-usdc-transfers", indexer, {
      fromBlock: 800_000n,
      toBlock: 800_005n,
    });

    const database = getTestDatabase(testResult);
```

where `testResult: { ctx: Context }`. `getTestDatabase` takes that argument and extracts the database from the context, casting the result to a database object.